<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RasterConverterDialogBase</class>
 <widget class="QDialog" name="RasterConverterDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Raster Converter</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Input file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Output file:</string>
   </property>
   <property name="buddy">
    <cstring>outputEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="savePathButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSlider" name="tolSlider">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Grouping tolerance:</string>
   </property>
   <property name="buddy">
    <cstring>tolSlider</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="valueLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="buddy">
    <cstring>tolSlider</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="advButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>361</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openPathButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>RasterConverterDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>RasterConverterDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
